src/tv2-common/actions/executeAction.ts

```diff
@@ -195,7 +195,8 @@ export async function executeAction<
 	coreContext: IActionExecutionContext,
 	settings: ActionExecutionSettings<StudioConfig, ShowStyleConfig>,
 	actionIdStr: string,
-	userData: ActionUserData
+	userData: ActionUserData,
+	triggerMode?: string
```

Comment:
Perhaps not here because we might add other actions that use other trigger modes unrelated to servers, but I changed it in `executeActionSelectServerClip` and further down.